core-bundle/src/DataContainer/DataContainerOperation.php

```diff
@@ -28,7 +28,7 @@ class DataContainerOperation implements \ArrayAccess
      */
     public function __construct(private readonly string $name, array $operation, private readonly array $record, private readonly DataContainer $dataContainer)
     {
-        $id = StringUtil::specialchars(rawurldecode((string) $record['id']));
+        $id = StringUtil::specialchars(rawurldecode((string) ($record['id'] ?? 0)));
```

Comment:
This looks strange to me. Should it be `(int)` instead of `(string)` or `?? '0'` instead of `?? 0`?

Reply:
anything works 🤷  I used `0` because the ID is usually numberic, but the string casting is there because `rawurldecode` expectes a string value. Feel free to change it though